.github/workflows/build.yml

```diff
@@ -0,0 +1,59 @@
+name: "Build / make"
```

Comment:
IMHO "Build" is sufficient, "make" is redundant here, since the commit status will look like "Build / make / make-check", "Build / make-check" is sufficient.